common/crypto/openssl/ec.c

```diff
@@ -560,7 +802,7 @@ bool oe_ec_valid_raw_private_key(
     if (bn != NULL)
         BN_clear_free(bn);
     if (group != NULL)
-        EC_GROUP_clear_free(group);
+        EC_GROUP_free(group);
```

Comment:
Is this a bug fix (for both OpenSSL 1.1.1 and 3)?

Reply:
EC_group_clear_free is no longer available in 3, and apparently EC_group_free is supposed to cover the same functionality.